include/freerdp/server/rail.h

```diff
@@ -130,7 +130,7 @@ struct _rail_server_context
 	psRailServerGetAppidRespEx ServerGetAppidRespEx;
 
 	RailServerPrivate* priv;
-	rdpContext* rdpContext;
+	rdpContext* rdpcontext;
 };
```

Comment:
This renaming sounds like a bad idea

Reply:
because?
it is a must because it causes a compilation error when mixing c and c++ code (typedef changes meaning of... blabla warning)

Reply:
agree with @hardening